include/nbl/ext/MitsubaLoader/CElementEmissionProfile.h

```diff
@@ -58,7 +57,7 @@ struct CElementEmissionProfile : public IElement {
 	std::string getLogName() const override { return "emissionprofile "; }
 
 	std::string filename;
-	float normalizeEnergy;
+	float normalizeEnergy, flatten;
```

Comment:
you're missing the logic/reg that loads the `flatten` from the XML, add it to a scene to test the parsing